src/main/scala/com/jcdecaux/setl/config/Conf.scala

```diff
@@ -217,8 +217,8 @@ object Conf {
     try {
       f(v)
     } catch {
-      case _: IllegalArgumentException => throw new ConfException.Format(s"Can't convert $v to $classOfT")
       case _: NumberFormatException => throw new ConfException.Format(s"Can't convert $v to $classOfT")
+      case _: IllegalArgumentException => throw new ConfException.Format(s"Can't convert $v to $classOfT")
```

Comment:
That's because a ```NumberFormatException``` is also a ```IllegalArgumentException```.
If ```IllegalArgumentException``` is caught first, the code will never reach ```NumberFormatException```. So the second case is for ```IllegalArgumentException``` that is not ```NumberFormatException```.